modules/client/test/integration/tokenVoting-client/methods.test.ts

```diff
@@ -1442,6 +1447,54 @@ describe("Token Voting Client", () => {
         const token = await client.methods.getToken(pluginAddress);
         expect(token).toBe(null);
       });
+
+      it("Should check if a ERC20 is compatible with governance and throw", async () => {
```

Comment:
```suggestion
      it("Should check if a ERC20 is compatible with the TokenVotingSetup and throw", async () => {
```
Change the other instances as well